br.gov.cti.invesalius.json

```diff
@@ -0,0 +1,216 @@
+{
+    "app-id": "br.gov.cti.invesalius",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.gfortran-62"],
+    "command": "runner.sh",
+    "build-options": {
+        "append-path": "/usr/lib/sdk/gfortran-62/bin"
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home",
+        /* For optical drive access and usb drivers*/
+        "--device=all"
```

Comment:
Does the app reads devices directly, at block level? If not, this is not needed.

Reply:
I just need to access the filesystem. Done!